Utils/Functions/System/virtual desktop/script.js

```diff
@@ -0,0 +1,19 @@
+var screenManager = Bean("screenManager");
```

Comment:
`const`

Reply:
Then why it's var in https://github.com/Zemogiter/Scripts/blob/3781c3e6a64c5f095b3fb8587607a771d5001163/Utils/Functions/Filesystem/Files/script.js#L1

Reply:
Because I forgot to change it to `const`

Reply:
okay ^^